firebase-sessions/src/main/kotlin/com/google/firebase/sessions/ProcessDataManager.kt

```diff
@@ -68,6 +85,11 @@ constructor(private val appContext: Context, private val uuidGenerator: UuidGene
       .all { (processDetails, processData) -> isProcessStale(processDetails, processData) }
   }
 
+  override fun isMyProcessStale(processDataMap: Map<String, ProcessData>): Boolean {
```

Comment:
AFAICT this isn't used anywhere except for unit tests.

Can you add a TODO / add usage?

Reply:
This will be used in the SharedSessionRepo in my next PR. This is following the design in the doc